tox.ini

```diff
@@ -23,7 +23,9 @@ envlist = lint, py{37, 38, 39}, docs, check-packaging
 
 [testenv]
 download = true
-deps = en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.0.0/en_core_web_sm-3.0.0.tar.gz
+deps =
+    en-core-web-sm @ https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.0.0/en_core_web_sm-3.0.0.tar.gz
+    en-core-sci-lg @ https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/releases/v0.4.0/en_core_sci_lg-0.4.0.tar.gz
```

Comment:
That's a chonky archive to download for tests (500MB), do you think it would be possible to structure our code so that we can swap `en_core_sci_lg` for `en_core_web_sm` in tests?

Reply:
I guess we could find a code structure allowing to do such swap but it would require much refactoring.

What do you suggest otherwise?

Reply:
So this model is used in `bluesearch/entrypoint/database/add.py::run`, and I just looked at the unit tests for this and I'm a bit surprised to see that we're calling `main` there and not `run`. I don't thinks it's coming from this PR, must've been introduced at some point earlier.

So, in `test_add.py` instead of 
```python
        args_and_opts = [
            "add",
            engine_sqlite.url.database,
            str(input_path),
            "--db-type=sqlite",
        ]

        main(args_and_opts)
```
I would probably have preferred
```python
run(db_url=engine_sqlite.url.database, path=str(input_path), db_type="sqlite")
```

In this case it would've been easy to parametrize the spacy model by adding a new parameter with a production default to `run`:
```python
def run(
    *,
    db_url: str,
    path: str,
    db_type: str,
    nlp_model_name: str = "en_core_sci_lg",
) -> None: ...
```
and then in unit tests one could put
```python
run(
    db_url=engine_sqlite.url.database,
    path=str(input_path),
    db_type="sqlite",
    nlp_model_name="en_core_web_sm",
)
```

Reply:
@Stannislav I think that the reason why I decided to go for `main("commandname", ...)` instead of `run_commandname(...)` is to also test that the parsing works. Which is a part of the CLI. However, we can change the logic:) Not sure which one makes more sense.

Reply:
I see I see. Like you said yourself, maybe it's a bit more in the spirit of unit tests to actually tests the functions separatly.

Like you said, maybe it's possible to write a separate test to check that `get_parser` and `run` match, maybe even without calling `run`, e.g. with the `inspect` module.

Reply:
Feel free to create an issue:))

Reply:
Issue: #446